packages/fields/src/fields/slug/slug-edit.tsx

```diff
@@ -27,7 +27,7 @@ const SlugEdit = ( {
 	field,
 	onChange,
 	data,
-}: DataFormControlProps< BasePost > ) => {
+}: DataFormControlProps< BasePost, false > ) => {
 	const { id } = field;
 
 	const slug = field.getValue( { item: data } ) || getSlug( data );
```

Comment:
If slug where to support bulk edits how would we handle the `getSlug` call here.
`field.getValue( { item: data } )` can be replaced by the `value` prop.